### DISCUSSION TITLE
使用Name的一次糟糕的体验

### DISCUSSION THREAD

**Discussion by giscus** [zu1k/blog#63] [Comments]
# 使用Name的一次糟糕的体验

A boy dreaming of traveling around the world.<br>This blog is used to record my study, thinking and life.

https://zu1k.com/posts/life/bad-experience-with-name/

**Comment by lizilong1993** [comment]
我买了个.xyz域名，3年的就花了十几块钱好像(❁´◡`❁)，你这有点倒霉2333
